packages/@sanity/cli/src/actions/init-project/initProject.ts

```diff
@@ -783,21 +973,19 @@ export default async function initSanity(
     return pFilter(organizations, (org) => hasProjectAttachGrant(org.id), {concurrency: 3})
   }
 
-  async function createOrAppendEnvVars(filename: string) {
+  async function createOrAppendEnvVars(
+    filename: string,
+    framework: Framework | null,
+    options?: {noLogs?: boolean}
```

Comment:
Code cleanup for later: negated flags are confusing - how about `log: false` instead?